### PR TITLE
Fix samples for AS policy rules

### PR DESCRIPTION
**Pull request by shantanusardal-okta**
## Description:
- **What's changed?** Fix samples for AS policy rules
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-340148](https://oktainc.atlassian.net/browse/OKTA-340148)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/api/authorization-servers/index.md [hunk 1]
```diff
@@ -1761,12 +1760,12 @@ Example from a Policy object
...
+| :----------- | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :------------------------------ | :---------------------------- |
+| clients      | For Policies, specifies which clients are included or excluded in the Policy                                                                                                                                                             | `include` and `exclude` lists   | True                          |
+| grantTypes   | Array of grantTypes this condition includes. Can include any of the following: `authorization_code`, `password`, `refresh_token`, or `client_credentials`. Determines the mechanism Okta uses to authorize the creation of the tokens.   | `include` list                  | True                          |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/authorization-servers/index.md:1766]:
```suggestion
| grantTypes   | Array of grantTypes that this condition includes. Accepted grantTypes: `authorization_code`, `password`, `refresh_token`, or `client_credentials`. Determines the mechanism Okta uses to authorize the creation of the tokens.   | `include` list                  | True                          |
```

File: packages/@okta/vuepress-site/docs/reference/api/authorization-servers/index.md [hunk 2]
```diff
@@ -1761,12 +1760,12 @@ Example from a Policy object
...
+| people       | For rules, specifies which Users and Groups are included or excluded in the rule                                                                                                                                                         | `include` and `exclude` lists   | True                          |
+| scopes       | Array of Scopes this condition includes                                                                                                                                                                                                  | `include` list                  | True                          |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/authorization-servers/index.md:1768]:
```suggestion
| scopes       | Array of Scopes that this condition includes                                                                                                                                                                                                  | `include` list                  | True                          |
```

**Review by susanharper-okta — APPROVED** [review]
two minor things - approving so I don't hold you up.

File: packages/@okta/vuepress-site/docs/reference/api/authorization-servers/index.md (hunk 1 above)
Comment (shantanusardal-okta) [reply packages/@okta/vuepress-site/docs/reference/api/authorization-servers/index.md]: done

File: packages/@okta/vuepress-site/docs/reference/api/authorization-servers/index.md (hunk 2 above)
Comment (shantanusardal-okta) [reply packages/@okta/vuepress-site/docs/reference/api/authorization-servers/index.md]: done

**Pull request merged**